README.md

```diff
@@ -1,2 +1,75 @@
-# RACTF Backend
-The backend API for the RACTF platform.
+<h1 align="center">RACTF Core</h1>
+
+<div align="center">
+  🚩⛳🏁🏳️‍🌈
+</div>
+<div align="center">
+  <strong>A robust CTF platform</strong>
+</div>
+<div align="center">
+  A <strong>Really Awesome</strong> framework for hosting Cyber-Security Capture the Flag events
+</div>
+
```

Comment:
Perhaps put CI status badges here?

Reply:
the ci status badges look dogshit, i tried them

Reply:
A demonstration of our automation and CI/CD is far more useful than some emoji

Reply:
yea i agree.


but the badges look like dogshit

Reply:
![Run Tests](https://github.com/ractf/core/workflows/Run%20Tests/badge.svg)

Looks fine